docs/contracts/functions.rst

```diff
@@ -132,16 +120,6 @@ statement::
 If you use an early ``return`` to leave a function that has return variables,
 you must provide return values together with the return statement.
 
-.. note::
-    You cannot return some types from non-internal functions, notably
-    multi-dimensional dynamic arrays and structs. If you enable the
-    ABI coder v2 by adding ``pragma abicoder v2;``
-    to your source file then more types are available, but
-    ``mapping`` types are still limited to inside a single contract and you
-    cannot transfer them.
-
-.. _multi-return:
-
```

Comment:
Why was this deleted? In fact undoing this would fix the CI.

Reply:
I didn't know what it was. It's my mistake.

Reply:
It's an anchor so that we can link to this section from some other part of the documentation. It's not part of the note above so removing it is not a good idea :)